### ISSUE TITLE
apxproof breaks fancyvrb macros

### ISSUE THREAD

**Issue by kbauer** [PierreSenellart/apxproof#26]
Currently, the `apxproof` package redefines the `VerbatimOut` environment in a manner, that breaks it for any use *other* than in `apxproof`. This seems quite likely to produce severe conflicts with other packages.

# How to reproduce 

The following file will compile as expected.
But if `apxproof` is imported, it fails.

```
\documentclass[a5paper,12pt]{article}

\usepackage{fancyvrb}

\begin{document}

Before VerbatimOut.

\begin{VerbatimOut}{\jobname.x}
First line.
Second line.
\end{VerbatimOut}

After VerbatimOut, before VerbatimInput.

\VerbatimInput{\jobname.x}

After VerbatimInput.

\end{document}
```

# Possible solutions

1. Defining a package-local version `axp@VerbatimOut`, instead of redefining the global `VerbatimOut` environment.
2. Using an entirely different mechanism for accumulating appendix contents; On modern hardware, an accumulation form `\appto{\axp@appendixcontents}{CONTENTS}` (or `\eappto` with `\expandonce{\BODY}` when using  `\NewEnviron`) should work too.

**Comment by PierreSenellart** [comment]
Yes, I am very aware of this issue, which is discussed among known issues in the documentation. The first solution is the obvious one, and I should be able to use `\let` commands so as not to have to redefine the entire behavior of the `verbatim` package. I am a bit wary of the second option since it means keeping in memory a potentially very large amount of content, but I may be a bit too conservative.

It should not be long to implement the first solution, I will try and do this in the next few days.

**Comment by PierreSenellart** [comment]
Fixed in 1e3759e. It was even easier than I thought, as `fancyvrb` is very modular.
